numpy/random/mtrand/generate_randint_helpers.py

```diff
@@ -0,0 +1,152 @@
+"""
+Helper file for generating the necessary helper functions
+for each `dtype` in `np.random.randint`. The function
+`gen_randint_helpers` is called right before Cythonization
+in NumPy setup.
+
+"""
+
+# npy_dt : Desired C API data type
+# npy_udt : Unsigned version of desired C
+#           API data type (could be identical)
+# npy_dt : Desired Python API data type
+
+template = """
+def _rand_{npy_dt}(low, high, size, rngstate):
```

Comment:
Well, then you should take it up with the person who actually wrote the doc-strings :smile: - the docstrings I imagine are for internal usage and comprehension since that's a lot of undocumented code if you leave them out.